src/selectors/models/users.js

```diff
@@ -0,0 +1,73 @@
+import { createSelector } from "reselect";
+import get from "lodash/fp/get";
+
+export const userByID = get(["users", "byID"]);
+
+export const makeGetUserByID = userid =>
```

Comment:
Whenever possible, let's use camelCase for variables. 
```suggestion
export const makeGetUserByID = userID =>
```